app/views/layouts/account_side_nav.html.erb

```diff
@@ -0,0 +1,33 @@
+<% content_for(:nav) do %>
+  <%= render 'accounts/nav_auth', enable_mobile_nav: true, greeting: @view_model.header_personalization %>
+<% end %>
+
+<% content_for :content do %>
+  <div class="grid-row grid-gap mb3">
+    <div class="desktop:grid-offset-3 desktop:grid-col-3">
+      <%= image_tag(
+        asset_path('user-access.svg'),
+        alt: '',
+      ) %>
+    </div>
+    <div class="desktop:grid-col-5">
+      <div>
+        <%= t('account.navigation.access_services') %>
+      </div>
+      <div>
+        <%= link_to t('account.navigation.learn_more'), MarketingSite.base_url %>
+      </div>
+    </div>
+  </div>
+  <div class="grid-row grid-gap">
+    <div class="desktop:grid-col-4">
+      <%= render 'accounts/side_nav' %>
+    </div>
+    <div class="desktop:grid-col-8">
+      <%= yield %>
+    </div>
+  </div>
+<% end %>
+
+<%= render 'accounts/mobile_nav' %>
+<%= render template: 'layouts/base', locals: { disable_card: true } %>
```

Comment:
I was observing some strange issues with the session timeout "Need more time?" modal not appearing correctly on the account dashboard ([screenshot](https://user-images.githubusercontent.com/1779930/93357902-d3828b80-f80e-11ea-9aca-ffe6941af9b3.png)). After some debugging, I noticed that with the arrangement of this markup, the source code of the page will output the mobile navigation markup outside (before) the `<html>` ([screenshot](https://user-images.githubusercontent.com/1779930/93357764-aafa9180-f80e-11ea-8005-4b39a1d40cd0.png)). I think the problem the issues with the modal are due to the fact that the doctype isn't being interpreted correctly by the browser.

I'm not sure what's the simplest way to achieve it, but I'd expect the markup of `accounts/mobile_nav` to be rendered within the `body` tag.

Reply:
😱 looking into this